### PR TITLE
Stub new methods

### PR DESCRIPTION
**Pull request by Insprill**
# Description
Seems like the Paper devs have had some fun recently.

# Checklist
The following items should be checked before the pull request can be merged.
- [ ] Unit tests added.
- [x] Code follows existing code format.

# Info on creating a pull request
- Make sure that unit tests are added which test the relevant changes.
- Make sure that the changes follow the existing code format.

# For maintainer
When a PR is approved, the maintainer should label this PR with `release/*`.

- If the PR fixes a bug in MockBukkit, update the patch version. (if the current version is `0.4.1`, the new version should be `0.4.2`)
- If the PR adds a new feature to MockBukkit, update minor version. (if the current version is `0.4.1`, the new version should be `0.5.0`)

Note that a PR that fixes an `UnimplementedOperationException` should be considered a new version and not a bugfix.

### CODE REVIEW

File: src/main/java/be/seeseemelk/mockbukkit/entity/SchoolableFishMock.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import org.jetbrains.annotations.Nullable;
+
+import java.util.UUID;
+
+/**
+ * Mock implementation of a {@link SchoolableFish}.
+ *
+ * @see FishMock
+ */
+public abstract class SchoolableFishMock extends FishMock implements SchoolableFish
```
Comment (thelooter) [inline src/main/java/be/seeseemelk/mockbukkit/entity/SchoolableFishMock.java:16]: This should be it's own PR

**Review by thelooter — CHANGES_REQUESTED** [review]
One small Change

File: src/main/java/be/seeseemelk/mockbukkit/entity/SchoolableFishMock.java (hunk 1 above)
Comment (Insprill) [reply src/main/java/be/seeseemelk/mockbukkit/entity/SchoolableFishMock.java:16]: SchoolableFish is new, so without it those methods would need to be duplicated on all the schoolable fish classes.

**Review by thelooter — APPROVED** [review]
